### PR TITLE
course team styling fix

### PR DESCRIPTION
**Pull request by tasawernawaz**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
  - [ ] Tag @ferdi or @pdpinch for review
- [ ] Migrations
  - [ ] Migration is backwards-compatible with current production code
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested
- [ ] Settings
  - [ ] New settings are documented and present in `app.json`
  - [ ] New settings have reasonable development defaults, if applicable

#### What are the relevant tickets?
https://trello.com/c/OZCcoiTR/25-course-team-styling-fixes

#### What's this PR do?
Fix styling on the course team tab page.

#### How should this be manually tested?
Goto any course team tab page under the program page.

#### Screenshots (if appropriate)
Screenshot 2020-02-19 at 5 57 44 PM

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@abdulkdawson please take a look at this.

**Comment by abdulkdawson** [comment]
@tasawernawaz this looks good. Thank you!

File: cms/templates/cms/course_team_tab_page.html [hunk 1]
```diff
@@ -8,136 +8,137 @@
...
+                <img class="member-image" src="{{ contributor.value.image.file.url }}" alt="" />
                 <div class="bio-content">
-                  <p class="bio">{{ administrator.value.bio }}</p>
+                  <p class="bio">{{ contributor.value.bio }}</p>
                 </div>
               </div>
               <div class="profile-content">
-                <span class="name">{{ administrator.value.name }}</span>
-                {{ administrator.value.title }}
+                <span class="name">{{ contributor.value.name }}</span>
```
Comment (annagav) [inline cms/templates/cms/course_team_tab_page.html]: Why this change and where is it reflected in the back end?

Comment (tasawernawaz) [reply cms/templates/cms/course_team_tab_page.html]: I think this is something because of moving `divs' inside a container, I can see any reflections of this change on the page.
